packages/fxa-shared/metrics/glean/fxa-ui-metrics.yaml

```diff
@@ -366,6 +367,23 @@ cached_login:
     expires: never
     data_sensitivity:
       - interaction
+  forgot_pwd_submit:
+    type: event
+    description: |
+      TODO: added in https://github.com/mozilla/fxa/pull/16668
```

Comment:
@chenba can you help me write this description?

Reply:
'User clicks on the "Forgot Password" Link.'